extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getPageFairBootstrapBottomBody.mustache

```diff
@@ -0,0 +1,24 @@
+<!-- Begin PageFair recovery -->
+<script type="text/javascript">
+	require([
+		'ext.wikia.adEngine.adContext',
+		'wikia.instantGlobals'
+	], function (adContext, instantGlobals) {
+		var context = adContext.getContext();
+
+		if (context.opts.pageFairRecovery === true && !window.wgUserName) {
```

Comment:
What do you think of getting rid of `window.wgUserName` check here? Doesn't `context.opts.pageFairRecovery` includes the logic already?